application/aggregator.py

```diff
@@ -0,0 +1,58 @@
+"""
+Aggregates the data from all databases into one CSV with minimal cutting/processing
+"""
+import sqlite3
+import csv
+import pandas as pd
+
+HEADERS = list(['track_id', 'title', 'song_id', 'release', 'artist_id', 'artist_mbid',
+                'artist_name', 'duration', 'artist_familiarity', 'artist_hotttnesss',
+                'year', 'lyrics'])
+
+DATA = []
+
+with open('../tagtraum_genre.cls', mode='r') as genre_map:
+    GENRE_READER = csv.DictReader(genre_map, delimiter='\t')
+    with sqlite3.connect('../musixmatch_lyrics.db') as lyrics_db:
+        LYRICS_C = lyrics_db.cursor()
+        WORDS_C = lyrics_db.cursor()
```

Comment:
Why are all your non const  variables all caps, it makes your code harder to read.